packages/eslint-plugin/configs/es5.js

```diff
@@ -73,30 +73,6 @@ module.exports = {
 				'!': true,
 			},
 		} ],
-		'valid-jsdoc': [ 'error', {
-			prefer: {
-				arg: 'param',
-				argument: 'param',
-				extends: 'augments',
-				returns: 'return',
-			},
-			preferType: {
-				array: 'Array',
-				bool: 'boolean',
-				Boolean: 'boolean',
```

Comment:
Did we lose most of these preferred types and tags with the transition? I only see "Object" type being enforced in the current configuration:

https://github.com/WordPress/gutenberg/blob/da54d11ed51db75d154f82f4f6ee972a9b1a4a2f/packages/eslint-plugin/configs/jsdoc.js#L72-L74

Reply:
This something I did?

I may have deleted those that were set to the default value. Hindsight, that may have been wrong because doing that may have loosened the enforcement.

Reply:
Hm, I guess it may work just as well still. I haven't tested all of the previous values, but I do see a warning (as expected) when trying to capitalize a `{String}` type.

Reply:
https://github.com/gajus/eslint-plugin-jsdoc#default-preferred-aliases